### ISSUE TITLE
warning with 1.1.0  when using unix socket connection with Ruby 2.3.0p0

### ISSUE THREAD

**Issue by kevinschmidt** [sensu-plugins/sensu-plugins-docker#30]
I'm getting the following warning when using the unix socket connection and Ruby 2.3.0p0:

```
root@server:~# /opt/sensu/embedded/bin/check-docker-container.rb -h /var/run/docker.sock containername
/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/net_http_unix-0.2.1/lib/net_x/http_unix.rb:35:in `connect_unix': Object#timeout is deprecated, use Timeout.timeout instead.
CheckDockerContainers OK
```

Tested on Ubuntu 14.04 with Sensu 0.23.3-1.

**Comment by eheydrick** [comment]
This is fixed in https://github.com/puppetlabs/net_http_unix/pull/1 yet to be merged.

**Comment by guikcd** [comment]
:+1:  i've live patched net_http_unix on the fly to avoid this warning too :)

**Comment by eheydrick** [comment]
Fixed via #41 and released in [1.1.3](https://github.com/sensu-plugins/sensu-plugins-docker/releases/tag/1.1.3) which is now on rubygems.
